content/api/api-events.md

```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
 alwaysopen: false
 pre: ""
 ---
-The events API allows you to consume events generated by CloudWisdom. You can either grab a list of events generated by CloudWisdom or grab an individual event by its ID. To explore CloudWisdom’s REST API and test some requests, go here.
-
-**Request Header**
-
-| Header Name | Header Value |
-|--------------|------------------|
-| Content-Type | application/json |
-
-## POST
-### POST to /ingest/events/{apiId}
-Creates an external event within CloudWisdom.
-
-```
-[
-  {
-    "source": "My Custom Datasource",
-    "title": "Server message 2",
-    "tags": [
-      {
-        "name": "Event Tag",
-        "value": "External"
-      }
-    ],
-    "type": "INFO",
-    "data": {
-      "elementId": "us-east-1:i-123a4567",
-      "level": "WARNING",
-      "message": "WARNING: Event is triggering!"
+
+## About the Events API
+
+
+## GET from /events
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/events/getEventsUsingGET_1" theme="info" >}} GET {{< /button >}} Use this endpoint to get details on events and their associated policies/elements.
+
+{{% expand "View Method Details." %}}
+
+### Parameters
```

Comment:
These parameters are optional as a user could just make a request to /events and get events for the past hour. We should make a note about that.